Source/Classes/ActionHandling/Link.swift

```diff
@@ -271,12 +271,13 @@ extension NantesLabel {
 extension NSAttributedString {
     func findExistingLinks() -> Set<NSTextCheckingResult> {
         var relinks: Set<NSTextCheckingResult> = []
-        enumerateAttribute(.link,
-                           in: NSRange(location: 0, length: length),
-                           options: []) { attribute, linkRange, _ in
-                            if let url = attribute as? URL {
-                                relinks.insert(NSTextCheckingResult.linkCheckingResult(range: linkRange, url: url))
-                            }
+        enumerateAttribute(.link, in: NSRange(location: 0, length: length), options: []) { attribute, linkRange, _ in
+            var url = attribute as? URL
```

Comment:
we should make this a bit cleaner with non-optional typing
```suggestion
            let url: URL
            if let urlAttribute = attribute as? URL {
                url = urlAttribute
            } else if let stringAttribute = attribute as? String, let urlAttribute = URL(string: stringAttribute) {
                url = urlAttribute
            } else { 
                return
            }
```